### ISSUE TITLE
Issue on Demo Notebook 1a

### ISSUE THREAD

**Issue by bradyferster** [PCMDI/pcmdi_metrics#831]
I have recently installed the pcmdi package to use at LOCEAN / IPSL on the Ciclad server, but have been running into the following issue (attached). Do you have any tips or advice to advance? There were no issues running Demo 0 and I am running Python 3.

![Screen Shot 2022-03-30 at 11 25 16](https://user-images.githubusercontent.com/51099991/160856776-a41fed10-7017-469a-9877-db67cfca002b.png)

**Comment by lee1043** [comment]
Hi @bradyferster, thank you for your feedback. 

By looking at the error message, we suspect the issue might be related using ipython (thus jupyter notebook) on Ciclad. Could you try run `pcmdi_compute_climatologies.py -p basic_annual_cycle_param.py` straight from command line in the same location and the same conda environment,  and see if that resolves your issue?

**Comment by bradyferster** [comment]
Hello @lee1043 
Thank you for the quick response.

I have tried running the command in bash as you suggested but have run into a different error message than the notebook.

I also tried to compile a new Python 3 environment on Ciclad and ran into the same issues, both in bash and jupyter notebook.

![Screen Shot 2022-03-31 at 15 08 58](https://user-images.githubusercontent.com/51099991/161062360-7ff65a84-f047-40d2-8059-3392bb1b8733.png)

**Comment by lee1043** [comment]
@bradyferster thank you for checking on this. Could you please share your basic_annual_cycle_param.py file with me? I'd like to check a few things in there.

**Comment by bradyferster** [comment]
@lee1043 Please find the attached basic_annual_cycle_param.py file

[basic_annual_cycle_param.py.zip](https://github.com/PCMDI/pcmdi_metrics/files/8395590/basic_annual_cycle_param.py.zip)

**Comment by lee1043** [comment]
@bradyferster I think we found the solution and we fixed it in https://github.com/PCMDI/pcmdi_metrics/pull/833 (thanks to @acordonez!). While we are releasing a minor updated version ([v2.2.2](https://github.com/PCMDI/pcmdi_metrics/releases/tag/v2.2.2)) that includes this fix, you can also make a quick fix as follow

```
which pcmdi_compute_climatologies.py
```
open the file
```
vi (yourpath)/pcmdi_compute_climatologies.py
```

And add `#!/usr/bin/env python` at the top

If you'd prefer quick fix, could you to see if that help your issue?

Otherwise I think you can update your PMP to the [new version](https://github.com/PCMDI/pcmdi_metrics/releases/tag/v2.2.2) -- I will keep you posted when the new version is ready on conda-forge.

**Comment by lee1043** [comment]
I believe this issue should have been resolved with v2.2.2. @bradyferster please feel free to reopen this issue if you still face problem.
